src/components/FeatureViews/Accounts/ProfileHeading.js

```diff
@@ -252,7 +248,16 @@ const ProfileHeading = ({ ownerID }) => {
                 </ResponsiveButton>
               )}
             </div>
-            <ResetDialog />
+            <ResetPasswordDialogs
```

Comment:
sorry, i should've clarified. also move all of the functions (handleClosePassword, handleClosePasswordReset, handleCloseReset) as well as all the related state variables into this component, so when anybody else looks at this component (ResetPasswordDialogs) they only need to look at 1 file to understand what's going on. You may need to include the "Reset Password" button into this component as well. 

so the component shuold look something like
```
const ResetPasswordDialogs = ({ //relevant props }) => {
    const [open, setOpen] = setState(false); 
    // other relevant state variables, graphql queries, and functions
    
    return (<>
          <ResponsiveButton {//relevant props}> Reset Password </ResponsiveButton>
          <Dialog {//relevant props}/> // step 1 dialog
          <Dialog {//relevant props}/> // step 2 dialog
    </>
}
```

Reply:
Got it. I've moved everything over, including the mutation